docs/manual-instrumentation.md

```diff
@@ -0,0 +1,43 @@
+# Manually instrument a .NET application
+
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches,
+you'll be able to present a more detailed representation of the logic
+and functionality of your application, clients, and framework.
+
+## Instrument using System.Diagnostics API
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
```

Comment:
```suggestion
For the list of steps required, see the [upstream documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
```

Reply:
Any step that would be custom for our distribution?

Reply:
No custom steps needed for our distribution.